manager/dispatcher/dispatcher.go

```diff
@@ -603,16 +607,26 @@ func (d *Dispatcher) Tasks(r *api.TasksRequest, stream api.Dispatcher_TasksServe
 					delete(tasksMap, v.Task.ID)
 					modificationCnt++
 				}
-			case <-time.After(eventPausedGap):
 				if modificationCnt > 0 {
```

Comment:
Removed the `if`.

We may have to readd it in the future if we add cases that don't always increment `modificationCnt`.